docs/pages/billing/faq.mdx

```diff
@@ -0,0 +1,119 @@
+
+### I am on a Free plan and only need a few extra builds
+
+If you are on a Free plan and have completed your monthly quota of free builds, upgrade to the [On-demand plan](https://www.notion.so/Subscriptions-Plans-and-Add-ons-3f928a4b5a944f918cd7cc5a17c91005?pvs=21). It offers a pay-as-you-go option for EAS Build and the same limits as the Free plan for EAS Update. Once your requirements are fulfilled, you can always downgrade to the Free from the On-demand plan.
+
+To upgrade from the Free to the On-demand plan, see [Upgrade to a new plan](/billing/manage/#upgrade-to-a-new-plan).
+
+To downgrade from an On-demand to a Free plan, see [Cancel a plan](/billing/manage/#cancel-a-plan).
+
+## Billing
+
+### What does a billing cycle start for a plan?
+
+For the Free plan, the billing cycle starts on the first day of the calendar month.
+
+For all other paid plans, the billing cycle starts from the subscription date of that plan.
+
+### How do I update my billing information?
+
+To update your Organization account's billing information, make sure that you have either [**Owner** or **Admin** role](/accounts/working-together/#manage-access). For a Personal account, you always have an **Owner** role. After verifying the role:
+
+- Go to your [**Account or Organization settings** > **Billing**](https://expo.dev/settings/billing), and click on **Update billing information**.
+- On Stripe's Portal, under **Billing information**, click on **Update information** to update billing-related information such as billing name, email, address, and tax ID Billing information.
+
+See [Manage billing information](/billing/manage/#manage-billing-information) for more details.
+
+### Can I update the billing info for my last invoice?
+
+No. Updating billing information will only be reflected on the next invoice.
+
+### Can you email me the receipts?
+
+No. Your account's Owner or Admin can download them. See [Download a receipt](/billing/invoices-and-receipts/#download-and-view-a-receipt) for more information.
+
+### How can I reduce the amount of EAS Build usage by using EAS Update?
+
+Use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without creating a new build. This option is better for most apps since JavaScript code changes more frequently than the underlying native code. You can create multiple test channels with EAS Update and reduce the need to create additional builds for your team. For native code changes, you can also automate the build process using [Expo Fingerprint](https://expo.dev/blog/fingerprint-your-native-runtime).
+
+For more information, see [How to optimize build usage](/billing/usage-based-pricing/#how-to-optimize-build-usage).
+
+### How do I estimate my next on-demand bill?
```

Comment:
Nit: let's remove `on-demand`. This answer applies for all bills. Let's also standardize language: `on-demand` should only apply to the plan and its customers, whereas `usage-based billing` or `overage billing` should be the norm for what we charge